<?xml version="1.0" encoding="utf-8"?>
<robot name="body" xmlns:xacro="http://www.ros.org/wiki/xacro">


<xacro:include filename="$(find surgical_sim)/urdf/body/templates/gazebo.links.xacro"/>

  <xacro:macro name="create_torso_links" params="parent_link xyz rpy">

    <!-- link 1: torso_link -->
    <link name="torso_link">
      <visual>
        <origin rpy="${-pi/2} 0 0" xyz="0 0 0"/>
        <geometry>
          <mesh filename="package://surgical_sim/model/meshes/dae/body/torso.dae"/>      
        </geometry>
      </visual>
      <!-- <inertial>
				<origin xyz="0 0 0" rpy="0 0 0"/>
				<mass value="1"/>
				<inertia ixx="1" ixy="0.0" ixz="0.0" iyy="1" iyz="0.0" izz="1"/>
			</inertial>  -->
    </link>

   <link name="/left/trocar_link"/>
   <link name="/right/trocar_link"/>

   <xacro:gazebo_links name="torso_link"/>
 
  </xacro:macro>
  

</robot>
